Apache-Arrow-Flight-Tester/main.py

```diff
@@ -90,6 +90,42 @@ def list_actions(flight_client):
     print(list(response))
 
 
+def update_object_store_to_s3(flight_client, endpoint, bucket_name, access_key_id, secret_access_key):
+    object_store_type_bytes = str.encode("s3")
+    object_store_type_size = len(object_store_type_bytes).to_bytes(2, byteorder="big")
+
+    arguments = [endpoint, bucket_name, access_key_id, secret_access_key]
+    action_body = object_store_type_size + object_store_type_bytes + create_update_object_store_action_body(arguments)
+
+    result = flight_client.do_action(pyarrow.flight.Action("UpdateRemoteObjectStore", action_body))
+
+    print(list(result))
+
+
+def update_object_store_to_azure_blob_storage(flight_client, account, access_key, container_name):
+    object_store_type_bytes = str.encode("azureblobstorage")
+    object_store_type_size = len(object_store_type_bytes).to_bytes(2, byteorder="big")
```

Comment:
Unclear why two bytes is used for the length instead of one when `len("s3") == 2` and `len("azureblobstorage") == 16` and 2<sup>8</sup> is 256

Reply:
Two bytes are used for the length so it is consistent with the way the arguments are encoded in the array of bytes.